readyplayerme/rpm-unity-sdk-core
# [SDK-454] feature/settings-handler-refactor

## [SDK-454](https://ready-player-me.atlassian.net/browse/SDK-454)

## Description

- refactored core settings handler to remove all editor logic
- also includes some other edits required to make this work
- added error logging for case where settings asset does not exist. 






## How to Test

-  install or check out this branch
- please check everything related to core settings loading, accessing, editing
- this includes changing subdomain, analytics and other settings from our various editor windows
- what happens if you delete the scriptable object does it break in editor or for runtime? 
- try building a sample scene and running, are there build or runtime errors? (related to core settings not loading or something) 
- when installing fresh does core settings get created as expected? 


## Checklist








[SDK-454]: https://ready-player-me.atlassian.net/browse/SDK-454?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
